### ISSUE TITLE
Update README.md to show python dependencies 

### ISSUE THREAD

**Issue by hackerpants** [cottonbeckfield/python-csr#1]
When I tried to run for the first time I got 

`ImportError: No module named OpenSSL`

`pip install pyOpenSSL` resolved it on Mac OS X, 

Perhaps update the README to show python dependencies?

**Comment by cjcotton** [comment]
Will do.

**Comment by hackerpants** [comment]
I just tried it on Ubuntu 14.04 too but looks like pyOpenSSL is part of the python distribution

So I guess its arguable that given the differences in OS and the way people manage dependencies with python that maybe you could expect the user to figure it out themselves?

**Comment by cottonbeckfield** [comment]
The OpenSSL library is in the Linux version of python already.

I would believe this to be an OS specific issue. I program on a Mac, and this particular script did not use virtualenv and I didn't have an issue with needing to install pyOpenSSL. Your version of python may not have been compiled with the OpenSSL support; https://docs.python.org/3.4/library/ssl.html

I don't mind making a note that it may be needed, but that might be digging a bit deep.

**Comment by hackerpants** [comment]
I looked a bit more, I see that there are two standard python openssl libraries, one is `ssl` the other is `pyOpenSSL`. 

My current python 2.7.10 was brew installed, the `import OpenSSL` doesn't work until I pip install pyOpenSSL, however I also have python 2.6.9 installed where `import OpenSSL` does work, so it appears to be down to python versions as you suggested

TL;DR
With standard python distributions, there doesn't seem to be a problem, if you want to use Python 2.7.10 in Mac OS X you need to `pip install pyOpenSSL`

**Comment by cottonbeckfield** [comment]
In a future update, I will have a note of what version of Python I used, and also some side FAQs for additional help for those who might be using alternative versions. 

Thanks for looking into it.
